store/storetest/plugin_store.go

```diff
@@ -97,14 +97,14 @@ func doTestPluginSaveOrUpdate(t *testing.T, ss store.Store, s SqlSupplier, doer
 			ExpireAt: expireAt,
 		}
 
-		retKV, err := ss.Plugin().SaveOrUpdate(kv)
+		retKV, err := doer(kv)
 		require.Nil(t, err)
 		assert.Equal(t, kv, retKV)
 		// SaveOrUpdate returns the kv passed in, so test each field individually for
 		// completeness. It should probably be changed to not bother doing that.
 		assert.Equal(t, pluginId, kv.PluginId)
 		assert.Equal(t, key, kv.Key)
-		assert.Equal(t, []byte(value), retKV.Value)
+		assert.Equal(t, []byte(value), kv.Value)
```

Comment:
Incorrectly testing against `retKV` here and below: the goal was just to verify that `kv` wasn't unexpectedly changed after calling in.